sale_packaging_price/views/product_view.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <record id="sale_packaging_price_product_tree_inherit" model="ir.ui.view">
```

Comment:
This is not a requirement, but just letting you know that ids get automatic prefix by the module name, so this xmlid will become `sale_packaging_price.sale_packaging_price_product_tree_inherit`, which is redundant as you see.